extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/mpmetrics/MetricDescriptor.java

```diff
@@ -10,7 +10,7 @@
 
 import io.micrometer.core.instrument.Tags;
 
-class MetricDescriptor {
+public class MetricDescriptor {
```

Comment:
Not completely sure about this, but without this change I am seeing this in the log on startup:
```
2022-01-06 09:26:55,793 WARN  [io.qua.arc.pro.Methods] (build-4) A method register() declared on io.quarkus.micrometer.runtime.binder.mpmetrics.AnnotatedGaugeAdapter$GaugeAdapterImpl has a non-public parameter of type io.quarkus.micrometer.runtime.binder.mpmetrics.MetricDescriptor which prevents it from being intercepted. Please change the parameter type visibility in order to make it intercepted.
```